neurobooth_os/main_control_rec.py

```diff
@@ -42,26 +38,6 @@ def start_servers(nodes=("acquisition", "presentation")):
         start_server(node)
 
 
-def prepare_feedback(nodes=("acquisition", "presentation")):
-    nodes = _get_nodes(nodes)
-    for node in nodes:
-        if node.startswith("acq"):
-            msg = "vis_stream"
-        elif node.startswith("pres"):
-            msg = "scr_stream"
-        else:
-            return
-    socket_message(msg, node)
-
-
-def prepare_devices(collection_id="mvp_025", nodes=("acquisition", "presentation")):
```

Comment:
Is this really not reachable? (I.e., is it a duplicate?) This resembles the message sent out during the "connect devices" phase.